src/main/resources/db/migration/V1_4__update_timestamp_with_time_zone.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE notification ALTER created_date_time TYPE TIMESTAMP WITH TIME ZONE;
```

Comment:
Do we want this script? Or can we just blow the whole database away and start fresh?